cmd/applications-rp/applications-rp-dev.yaml

```diff
@@ -8,7 +8,7 @@
 # - Disables metrics and profiler
 #
 environment:
-  name: self-hosted
```

Comment:
This is another place where I made things more consistent. Now all of the `dev` files say `dev` and are named with `dev`.